calico/assets/monitors/dataplane_failures.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Calico dataplane failure",
```

```suggestion
  "name": "[Calico] dataplane failures",
```

Integration name should be in `[]`